source/language/gates.rst

```diff
@@ -103,9 +103,126 @@ definitions for which :ref:`defcal blocks <pulse-gates>` are known. A more sophi
 might use the `gate` definitions of the gates with associated ``defcal`` blocks to
 build a gate library, and use methods based on KAK decompositions to rewrite into this hardware library.
 
-The ``gate`` statement also allows defining parameterized families of unitaries. For example, a CPHASE
-operation is shown schematically in :numref:`fig_gate`
-and the corresponding OpenQASM code is
+Controlled gates can be constructed by adding a control modifier to an existing gate. For example,
```

Comment:
Prior to this change set, the discussion of controlled gates sat under a "quantum gate modifiers" section. I'd suggest preserving that organizational structure.

Reply:
Actually, this entire section of looks like it refers to a stale version of the "quantum gate modifiers" section.

Reply:
Agreed. For example, the broadcasting discussion below is already discussed adequately elsewhere https://openqasm.com/language/gates.html#broadcasting.